guidance/virtual_reproducibility_verification_protocol.md

```diff
@@ -0,0 +1,124 @@
+# Virtual Reproducibility Verification Protocol
+
+This protocol outlines the steps for conducting a **Virtual Reproducibility Verification** session. This approach is used when the reproducibility package relies on strictly confidential data that cannot be shared with the replication team.
+During this session, the author executes the full reproducibility package in a clean environment while the reviewer observes virtually. The aim is to ensure that the code runs successfully and produces the expected outputs, all while documenting and preserving a clean and reproducible setup.
+
+---
+
+## 🔁 General Procedure
+
+1. **Schedule a Virtual Session**  
+   - The reviewer and author schedule a session via Microsoft Teams (or similar), with screen sharing enabled.
+
+2. **Generate Dataset Hashes (Before Running the Code)**  
+   - Use the official [hashing tool](https://github.com/worldbank/wb-reproducible-research-repository/tree/main/resources/generate_hash) provided by the reproducibility team.  
+   - To run the tool, authors must have R and RStudio installed on their computer.  
```

Comment:
```suggestion
   - To run the tool, authors must have R installed on their computer.  
```
RStudio is not strictly needed. I had a meeting yesterday where we only installed R and did it through the R GUI and it worked.